packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5979,6 +5983,27 @@ export default class Meeting extends StatelessWebexPlugin {
     return !this.isRoapInProgress;
   }
 
+  /**
+   *  media failed, so collect a stats report from webrtc using the wcme connection to grab the rtc stats report
+   * send a webrtc telemetry dump to the configured server using the internal media core check metrics configured callback
+   * @param {String} callFrom - the function calling this function, optional.
+   * @returns {Promise<void>}
+   */
+  private forceSendStatsReport = async ({callFrom}: {callFrom?: string}) => {
+    const LOG_HEADER = `Meeting:index#forceSendStatsReport --> called from ${callFrom} : `;
+    try {
+      await this.mediaProperties?.webrtcMediaConnection?.forceRtcMetricsSend();
+      LoggerProxy.logger.info(
+        `${LOG_HEADER} failed but successfully uploaded available webrtc telemetry statistics`
+      );
+    } catch (e) {
+      LoggerProxy.logger.error(
+        `${LOG_HEADER} failed and failed to upload webrtc telemetry statistics: `,
```

Comment:
same here

Reply:
oh and we will always get this error for non-multistream meetings, because forceRtcMetricsSend() doesn't exist in RoapMediaConnection class, but I guess that's ok, the log tells the truth that we didn't upload the dump in that case

Reply:
yeah thats what I was aiming for. and removed the success/failure comments from them.